contracts/test/vault/index.js

```diff
@@ -531,4 +539,35 @@ describe("Vault", function () {
       vault.connect(governor).setVaultBuffer(utils.parseUnits("2", 19))
     ).to.be.revertedWith("Invalid value");
   });
+
+  it("Should only allow Governor and Strategist to call withdrawAllFromStrategies", async () => {
+    const { vault, governor, matt, strategist } = await loadFixture(
+      defaultFixture
+    );
+    await vault.connect(governor).withdrawAllFromStrategies();
+    await vault.connect(strategist).withdrawAllFromStrategies();
+    await expect(
+      vault.connect(matt).withdrawAllFromStrategies()
+    ).to.be.revertedWith("Caller is not the Strategist or Governor");
+  });
+
+  it("Should only allow Governor and Strategist to call withdrawAllFromStrategy", async () => {
+    const {
+      vault,
+      governor,
+      strategist,
+      compoundStrategy,
+      matt,
+    } = await loadFixture(defaultFixture);
+    await vault.connect(governor).approveStrategy(compoundStrategy.address);
+    await vault
+      .connect(governor)
+      .withdrawAllFromStrategy(compoundStrategy.address);
```

Comment:
Maybe add some assets to the Compound strategy and ensure they get withdrawn?

Reply:
Still on my TODO list. I did not forget...